Sample App/WatchKit Sample App Extension/InterfaceController.swift

```diff
@@ -33,7 +33,7 @@ class InterfaceController: WKInterfaceController {
             self?.selectedTimeLabel.setText(timeFormatter.string(from: selectedTime))
         }
         
-        timePickerDataSource.setup(withInitiallySelectedDate: Date())
+        timePickerDataSource.updateDate(to: Date())
```

Comment:
Could you update the `README.md` to reflect this change? Otherwise, it looks good :)

Reply:
Sure.